nextcord/ui/view.py

```diff
@@ -160,6 +160,13 @@ class View:
         Whether or not to automatically defer the component interaction when the callback
         completes without responding to the interaction. Set this to ``False`` if you want to
         handle view interactions outside of the callback.
+    prevent_update: :class:`bool` = True
```

Comment:
Usually the default is mentioned in the actual documentation and not by the name. In other words:
```suggestion
    prevent_update: :class:`bool`
```

Reply:
What do you mean? I don't get it. The other docs show the default there as well.

Reply:
This is best left for another PR since the surrounding documentation in this module uses the same style for parameter defaults.